.gitignore

```diff
@@ -48,4 +48,9 @@ node_modules
 /node_modules
 yarn-debug.log*
 .yarn-integrity
-/yarn-error.log
\ No newline at end of file
+/yarn-error.log
+# Ignore application configuration
+/config/application.yml
+
+/config/master.key
+/config/credentials.yml.enc
```

Comment:
This file is encrypted and you like to store it in the repository!
https://blog.saeloun.com/2019/10/10/rails-6-adds-support-for-multi-environment-credentials.html